__tests__/__unit__/USSTree.unit.test.ts

```diff
@@ -383,7 +389,109 @@ describe("Unit Tests (Jest)", () => {
         await testTree.ussFilterPrompt(testTree.mSessionNodes[1]);
         expect(showInformationMessage.mock.calls.length).toBe(1);
         expect(showInformationMessage.mock.calls[0][0]).toBe("No selection made.");
+        theia = false;
```

Comment:
This was to put it back into VSCode only mode. It may not have been necessary outside of this particular it section but I didn't circle back to try it.